README.md

```diff
@@ -49,7 +49,65 @@ devtools::install_github("UBC-MDS/feature-selection-r")
 
 ## Usage
 
-- TODO
+To guide you with an example of how to use this package, we would use the [Friedman dataset](https://www.rdocumentation.org/packages/tgp/versions/2.4-14/topics/friedman.1.data).
+
+Load dataset:
+```
+data <- dplyr::select(tgp::friedman.1.data(), -Ytrue)
+X <- data[1:(length(data)-1)]
+y <- data[length(data)]
+```
+
+Use of feature selection functions:
+
+- forward_selection
+```
+# create a 'scorer'
+custom_scorer <- function(data){
+  model <- lm(Y ~ ., data)
+  return(mean(model$residuals^2))
+}
+
+# use function
+featureselection::forward_selection(custom_scorer, X, y, 3, 7)
+[1] 4 2 1 5
+```
+
+- recursive_feature_elimination
+```
+# create a 'scorer'
```

Comment:
Suggesting a more descriptive explanation:

```
# Create a custom scorer that returns the name of the column with the lowest coefficient weight.
```

Reply:
Good idea. I will add it as an overall comment at the beginning of the usage section.

Reply:
This comment is specific to the custom scorer needed for `recursive_feature_elimination`. The scorer used in `forward_selection` is returning MSE, for example, so add the appropriate description for each custom scorer.

Reply:
Ok, I will. Also I will put the message you give for your scorer.

Reply:
Ok, I will put that comment to the recursive_feature_elimination scorer.

Reply:
done